### ISSUE TITLE
How to enjoy Cosmic Background?

### ISSUE THREAD

**Issue by Thatoo** [lilyinstarlight/nixos-cosmic#302]
When I go to settings to choose between backgrounds, I do't have default Cosmic bg, I can choose a "color" background or one of my pictures but there is not official cosmic bg available.

**Comment by drakon64** [comment]
I don't believe the official background are packaged, but you can download them from https://github.com/pop-os/cosmic-wallpapers/tree/master/original and 
https://github.com/pop-os/wallpapers/tree/master/original.

**Comment by lilyinstarlight** [comment]
Some work is needed to get upstream cosmic-bg stuff to work with backgrounds from XDG_DATA_DIRS instead of just hardcoding `/usr/share/backgrounds`

**Comment by edrex** [comment]
For now, `Settings > Desktop > Wallpaper > Open New Folder` click the path edit button, paste `

**Comment by Thatoo** [comment]
> For now, `Settings > Desktop > Wallpaper > Open New Folder`, browse to `/run/current-system/sw/share/backgrounds/cosmic/`. Seems like cosmic apps have a separate clipboard from other apps currently ([pop-os/cosmic-epoch#1009](https://github.com/pop-os/cosmic-epoch/issues/1009)).

I guess that if the content of `/run/current-system/sw/share/backgrounds/cosmic/` could be accessible by default on fresh install that would be nice. Maybe it isn't doable and in this case, this issue could be closed.
